### DISCUSSION TITLE
Configuring "tailwind-merge"

### DISCUSSION THREAD

**Discussion by tylerforesthauser** [barvian/fluid-tailwind#29] [Help] [answered]
Hey hey,

I was actually just wondering if someone might have an idea of how to go about configuring "tailwind-merge" to allow easy overriding of both fluid and non-fluid classes. For example: 

- `~text-base/lg sm:~sm:~text-lg/xl ~text-2xl/3xl sm:~sm:~text-3xl/4xl` -> `~text-2xl/3xl sm:~sm:~text-3xl/4xl`
- `text-base sm:text-lg ~text-2xl/3xl sm:~sm:~text-3xl/4xl` -> `~text-2xl/3xl sm:~sm:~text-3xl/4xl`

**Comment by firxworx** [comment]
I haven't tried it with fluid.tw yet but I have used custom configurations for my own custom classes and utilities that I add via presets/plugins.

You can look at https://github.com/dcastil/tailwind-merge/blob/v2.3.0/docs/api-reference.md#extendtailwindmerge

Off-hand I am wondering if the `extend.classGroups` with a function would work to add `~` as a prefix and also provide patterns for the slash syntax `x/y`.

I am going to file an issue as a feature-request. I think it would be amazing to have official support (one can dream!).

Update: linking issue -- https://github.com/barvian/fluid-tailwind/issues/36

**Comment by barvian** [comment, marked as answer]
I just pushed an [official tailwind-merge plugin for `fluid-tailwind`](https://fluid.tw/#tailwind-merge) 🙂

**Reply by bumpyy** [reply]
Is there any way to integrate it with https://github.com/gehrisandro/tailwind-merge-laravel ?

**Reply by firxworx** [reply]
Confirming I tried `fluid-tailwind/tailwind-merge` plugin and that it is _awesome_.
Thank-you @barvian your work on this project is amazing. I'm going to standardize on it as part of my tailwind workflow going forward.
